frontend/src/store/account/actions.ts

```diff
@@ -13,4 +13,12 @@ const actions = {
   }
 }
 
+export function getMyBalance(): Promise<number> {
+  return new Promise(function (resolve, reject) {
+    api.getMyBalance(({ balance }: { balance: string }) => {
```

Comment:
Does this function ever return an error? The promise will resolve but never reject.